spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
```

Comment:
Only this require should be here..
The other one should be import in your rails_helper